src/controller/file_manager.controller.ts

```diff
@@ -36,4 +47,31 @@ export class FileManagerController {
 
     return this.fileManagerFormatter.toFolderResponse(folder);
   }
+
+  @Post('folder/delete')
+  @HttpCode(HttpStatus.OK)
+  @ApiBearerAuth('authorization')
+  @ApiResponse({ status: HttpStatus.OK, type: DeleteFolderResponse })
+  public async deleteFolder(@Req() req: Request, @Body() body: DeleteFolderRequest): Promise<DeleteFolderResponse> {
+    const user = req.raw.user;
+    return this.fileManagerService.deleteFolder(user, body);
```

Comment:
It's better to return some excessive info from service method and then use a formatter.